pkg/lib/web/static_asset_resolver.go

```diff
@@ -102,6 +67,14 @@ func (r *StaticAssetResolver) StaticAssetURL(id string) (string, error) {
 	return staticAssetURL(r.Config.PublicOrigin, string(r.StaticAssetsPrefix), hashPath)
 }
 
+func (r *StaticAssetResolver) GeneratedStaticAssetURL(key string) (string, error) {
+	desc, _ := StaticAssetResources["generated-asset"]
+
+	manifest := desc.(*GeneratedAssetDescriptor).manifest.Load().(*ManifestContext).content
```

Comment:
Instead of accessing the private field `manifest`, expose a method on GeneratedAssetDescriptor `GetAssetPathForKey(key string) (string, error)`. Propagate the error.